ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op.hpp

```diff
@@ -166,6 +231,16 @@ Tensor optimized_conv_new(const Tensor& a, const Tensor &b, std::optional<const
     bool use_non_tile_height = false
 );
 
+std::pair<uint32_t,uint32_t> estimate_L1_usage(
```

Comment:
can you add a comment with what exactly is being returned by this function.
I would prefer a small struct with named members instead of std::pair

Reply:
Done.